server/channels/db/migrations/mysql/000110_update_vacuuming.down.sql

```diff
@@ -0,0 +1 @@
+-- Only applicable to Postgres
```

Comment:
Out of curiosity, would MySQL deployments benefit from periodic `OPTIMIZE TABLE` instead?

Reply:
I don't think so. First of all, `OPTIMIZE TABLE` takes super long :P and I rarely have seen any benefit running them.

Reply:
Fair enough, had to ask :)